### DISCUSSION TITLE
Set upper limit key in intervalToDuraition function

### DISCUSSION THREAD

**Discussion by shaifulborhan** [js-fns/date-fns#2353] [Q&A]
When using the intervalToDuration() function, the duration results are returned in this format:

```
{
    "years": 0,
    "months": 0,
    "days": 0,
    "hours": 0,
    "minutes": 0,
    "seconds": 0
}
```

Is there a way to set the upper limit key so that the remaining duration value are not calculated beyond that key?

For example, if my duration results is this:

```
{
    "years": 0,
    "months": 0,
    "days": 1,
    "hours": 1,
    "minutes": 10,
    "seconds": 40
}
```

If I set `hours` as the upper limit key, I would expect the results to look like this:

```
{
    "years": 0,
    "months": 0,
    "days": 0,
    "hours": 25,
    "minutes": 10,
    "seconds": 40
}
```

Is there a way to accomplish something like this? At the moment I am doing manual calculation to hours if the days, months and years values are present.

Thank you!

**Comment by codingedgar** [comment]
Seems like https://date-fns.org/v2.19.0/docs/formatDistanceStrict might be closer, but I can't find anything like moment's.
